source/renderer/app/components/wallet/skins/AmountInputSkin.scss

```diff
@@ -20,5 +20,5 @@
   position: absolute;
   right: 20px;
   text-transform: uppercase;
-  user-select: text;
+  user-select: none;
```

Comment:
@DeeJayElly why did we add this?

Reply:
Text should be non selectable according to -> https://docs.google.com/document/d/1VuXLJ-IysCXUuMUkEHFABCCKAMo0OCJEzVIwLoCbKgs/edit?disco=AAAAJP3XFi0

Reply:
Got it - thanks!